src/styles/cylc/_tree.scss

```diff
@@ -55,14 +55,20 @@ $icon-size: 1.2em;
   }
 }
 
-.treeitem {
+.c-treeitem {
   display:inline-block;
   vertical-align:middle;
   width: 100%;
   .node {
     line-height: $line-height;
     display: block;
 
+    .node-expand-collapse-button {
+      width: 1.3em;
+      height: 1.3em;
```

Comment:
Alignment is slightly off

![image](https://github.com/cylc/cylc-ui/assets/61982285/7c4a7afe-cebb-44c2-833f-58c488ad9bd9)

This fixes it:
```suggestion
      width: 1.5em;
      height: 1.5em;
      margin: 0 2px;
```